btcec/ciphering.go

```diff
@@ -0,0 +1,192 @@
+package btcec
```

Comment:
We've recently relicensed everything "The btcsuite authors".  Please update the license accordingly.